src/iconv.cr

```diff
@@ -8,7 +8,7 @@ struct Iconv
     original_from, original_to = from, to
 
     @skip_invalid = invalid == :skip
-    {% unless flag?(:freebsd) %}
+    {% unless flag?(:freebsd) || flag?(:musl) %}
```

Comment:
Ah yes, I didn't see the __iconv usage for freebsd. I guess silently failing makes sense here.